[IMP] Productivity: Discuss Getting Started (#4645)
odoo/documentation: content/applications/productivity/discuss/overview/team_communication.rst

======================================
Efficiently communicate using channels
======================================

Channels can be used to organize discussions between individual teams, departments, projects, or any
other group that requires regular communication. This way, everyone in the loop updated with the
latest developments.

Public and private channels
===========================

> A *pulic* channel can be seen by everyone, while a *private* one is only visible to users invited to
> it.

Reviewer: Typo
```suggestion
A *public* channel can be seen by everyone, while a *private* one is only visible to users invited
to it.
```

> .. image:: team_communication/create_channel.png

Reviewer: Update filename please
```suggestion
.. image:: team_communication/create-channel.png
```

Also please update the alt tag for this image:

>    :alt: View of the Discuss sidebar and a channel being created in Odoo Discuss.

> Configuration options

Reviewer: Line 22-23 Early line break (I cant make a suggestion there)

```
   announcements), whereas a private channel could be used whenever information should be limited to
   specific groups (such as a specific department).
```

---------------------

> A channel’s name, description, email alias, and privacy can be configured by clicking on the
> *channel settings* ' ⚙️ ' icon on the sidebar.

Reviewer: Icon formatting
```suggestion
A channel's name, description, email alias, and privacy can be configured by clicking on the channel
settings :guilabel:`⚙️ (gear)` icon on the sidebar.
```

.. image:: team_communication/channel-settings.png
   :align: center
   :alt: View of a channel’s settings form in Odoo Discuss

> Privacy and Members

Reviewer: I _think_ members should be lowercase here
```suggestion
Privacy and members
```
